docs/components/input-controller/gpio.md

```diff
@@ -141,11 +128,11 @@ Click **Create component** and then fill in the attributes for your model:
+| `axes` | object | **Required** | The [Axes](../#axis-controls) available for control. These should be connected to the GPIO/ADC board. <pre> **Each axis's fields:** <br><br> <code>name</code>: Name of the Analog Reader that reports ADC values for the axis Control, as configured on the board component. <br><br> <code>control</code>: The [Control](../#control-field) type to use when reporting events as this axis's position is changed. <br><br> <code>min</code>: The minimum ADC value that the analog reader can report as this axis's position changes. <br><br> <code>max</code>: The maximum ADC value that the analog reader can report as this axis's position changes. <br><br> <code>deadzone</code>: The absolute ADC value change from the neutral `0` point to still consider as a neutral position. This option is given so tiny wiggles on loose controls don't result in events being reported. <br><br> <code>min_change</code>: The minimum absolute ADC value change from the previous ADC value reading that can occur before reporting a new [`PositionChangeAbs Event`](../#event-object). This option is given so tiny wiggles on loose controls don't result in events being reported. <br><br> <code>bidirectional</code>: Boolean indicating if the axis changes position in 1 direction, like on an analog trigger or pedal, or 2, like on an analog control stick. <pre> *true:* The axis should report center, `0`, as halfway between the min/max ADC values. <br> *false:* `0` is still the neutral point of the axis, but only positive change values can be reported. </pre> <code>poll_hz</code>: How many times per second to check for a new ADC reading that can generate an event. <br><br> <code>invert</code>: Boolean indicating if the direction of the axis should be flipped when translating ADC value readings to the axis Control value indicating position change. <pre> *true:* flips the direction of the axis so that the minimum ADC value is reported as the maximum axis Control value. <br> *false:* keeps the direction of the axis the same, so that the minimum ADC value is reported as the minimum axis Control value. </pre></pre>|
```

Comment:
```suggestion
| `axes` | object | **Required** | The [axes](../#axis-controls) available for control. These should be connected to the GPIO/ADC board. <pre> **Each axis's fields:** <br><br> <code>name</code>: Name of the Analog Reader that reports ADC values for the axis Control, as configured on the board component. <br><br> <code>control</code>: The [Control](../#control-field) type to use when reporting events as this axis's position is changed. <br><br> <code>min</code>: The minimum ADC value that the analog reader can report as this axis's position changes. <br><br> <code>max</code>: The maximum ADC value that the analog reader can report as this axis's position changes. <br><br> <code>deadzone</code>: The absolute ADC value change from the neutral `0` point to still consider as a neutral position. This option is given so tiny wiggles on loose controls don't result in events being reported. <br><br> <code>min_change</code>: The minimum absolute ADC value change from the previous ADC value reading that can occur before reporting a new [`PositionChangeAbs Event`](../#event-object). This option is given so tiny wiggles on loose controls don't result in events being reported. <br><br> <code>bidirectional</code>: Boolean indicating if the axis changes position in 1 direction, like on an analog trigger or pedal, or 2, like on an analog control stick. <pre> *true:* The axis should report center, `0`, as halfway between the min/max ADC values. <br> *false:* `0` is still the neutral point of the axis, but only positive change values can be reported. </pre> <code>poll_hz</code>: How many times per second to check for a new ADC reading that can generate an event. <br><br> <code>invert</code>: Boolean indicating if the direction of the axis should be flipped when translating ADC value readings to the axis Control value indicating position change. <pre> *true:* flips the direction of the axis so that the minimum ADC value is reported as the maximum axis Control value. <br> *false:* keeps the direction of the axis the same, so that the minimum ADC value is reported as the minimum axis Control value. </pre></pre>|
```